dbhydra/dbhydra_core.py

```diff
@@ -291,6 +290,7 @@ def __init__(self, config_file="config.ini", db_details=None):
 
         self.lock = threading.Lock()
 
+        # NOTE: Pointless call (?): This is simply returning ContextManager object as `self.connect_to_db()` is a contextmanager decorated
```

Comment:
It is not pointless call - it is testing of connection

Reply:
How is this testing connection? When connection is tested on platform, context manager `connect_to_db` method is used.  Here it's simply returning ContextManager object, which does not even get consumed.